src/view/editor/parts/editor-availability-warning-row.test.tsx

```diff
@@ -0,0 +1,476 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { faker } from '@faker-js/faker';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { screen } from '@testing-library/react';
+import { map } from 'lodash';
+import moment from 'moment';
+import { CALENDAR_RESOURCES } from '../../../constants';
+
+import {
+	EditorAvailabilityWarningRow,
+	getIsBusyAtTimeOfTheEvent
+} from './editor-availability-warning-row';
+import { setupTest } from '../../../carbonio-ui-commons/test/test-setup';
+import { generateEditor } from '../../../commons/editor-generator';
+import { reducers } from '../../../store/redux';
+
+const dateFormat = 'YYYY/MM/DD';
+
+describe('editor availability warning row', () => {
+	describe('getIsBusyAtTimeOfTheEvent', () => {
+		test('When an attendee is not busy at the time of the event its chip wont show a warning icon', () => {
```

Comment:
The test (and all the following tests) description is misleading.
It is mentioning an icon inside the chip but the test is testing only the boolean result of the _getIsBusyAtTimeOfTheEvent_ function